### PR TITLE
Minor updates to main and benchmark README files

### PR DESCRIPTION
**Pull request by bobbinth**
This PR updates some wording and benchmark tables in README files.

### CODE REVIEW

File: benches/README.md [hunk 1]
```diff
@@ -1,52 +1,49 @@
...
-
-* **BLAKE3:**
-As specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented in this crate.
-
-* **SHA3:**
-As specified [here](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf) and implemented [here](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/sha/mod.rs).
-
-* **Poseidon:**
-As specified [here](https://eprint.iacr.org/2019/458.pdf) and implemented (in pure Rust, without vectorized instructions) [here](https://github.com/mir-protocol/plonky2/blob/main/plonky2/src/hash/poseidon_goldilocks.rs).
+* **BLAKE3** as specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented [here](https://github.com/BLAKE3-team/BLAKE3) (with a wrapper exposed via this crate).
```
Comment (itzmeanjan) [inline benches/README.md:4]: Nothing major, probably we can pin BLAKE3 specification to specific commit.

File: benches/README.md [hunk 2]
```diff
@@ -1,52 +1,49 @@
...
+* **SHA3** as specified [here](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf) and implemented [here](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/sha/mod.rs).
+* **Poseidon** as specified [here](https://eprint.iacr.org/2019/458.pdf) and implemented [here](https://github.com/mir-protocol/plonky2/blob/main/plonky2/src/hash/poseidon_goldilocks.rs) (but in pure Rust, without vectorized instructions).
```
Comment (itzmeanjan) [inline benches/README.md:6]: Here also we can probably pin Poseidon implementation to specific commit.

**Review by itzmeanjan — APPROVED** [review]
I've added two minor comments, but I'm approving.

**Review by Al-Kindi-0 — APPROVED** [review]
Looks good, thank you!

File: benches/README.md (hunk 1 above)
Comment (bobbinth) [reply benches/README.md:4]: I think it's probably OK to keep as is - I'm hoping BLAKE3 specs won't change much.

File: benches/README.md (hunk 2 above)
Comment (bobbinth) [reply benches/README.md]: Agreed! Updated.

**Pull request merged**